pkg/reconciler/subscription/subscription.go

```diff
@@ -425,7 +423,7 @@ func isNilOrEmptyDestination(destination *duckv1.Destination) bool {
 
 func (r *Reconciler) syncPhysicalChannel(ctx context.Context, sub *v1.Subscription, channel *eventingduckv1.Channelable, isDeleted bool) (bool, error) {
 	logging.FromContext(ctx).Debugw("Reconciling physical from Channel", zap.Any("sub", sub))
-	if patched, patchErr := r.patchSubscription(ctx, sub.Namespace, channel, sub); patchErr != nil {
+	if patched, patchErr := r.updateChannelable(ctx, sub.Namespace, channel, sub); patchErr != nil {
```

Comment:
Good! 👍

Did you forget to change the variable name `patched` `patchErr`? 😁

Reply:
Ah, I missed it. Thank you. Subscription is immutable, so I thought that it is okay to compare only the length.